core/src/main/java/hudson/model/MyViewsProperty.java

```diff
@@ -254,7 +254,23 @@ public String getIconFileName() {
 		public String getUrlName() {
 			return "/me/my-views";
 		}
-		
+
+    }
+
+    private static class NewViewAction implements Action {
+        @Override
+        public String getIconFileName() {
+            return "new-document.png";
```

Comment:
Clashes with use of `folder` in https://github.com/jenkinsci/jenkins/pull/3086.

Reply:
ok, I'll change it to folder
but ```new-document``` seems more natural for me

Reply:
I do not have any particular opinion, just noting that two PRs seem to be proposing conflicting UIs.